lib/mastodon/media_cli.rb

```diff
@@ -224,10 +227,14 @@ def refresh
         scope = MediaAttachment.where(account_id: account.id)
       elsif options[:domain]
         scope = MediaAttachment.joins(:account).merge(Account.by_domain_and_subdomains(options[:domain]))
-      else
+      elsif options[:days].blank?
```

Comment:
Seems like with this you intend for `--days` to possibly be passed on its own, without `--status`, `--account` or `--domain`, but if that's the case, wouldn't `scope` be undefined?

Reply:
Yes, my mistake. I'll fix it 👍🏼